app/assets/javascripts/discourse/app/components/search-menu.js

```diff
@@ -108,9 +108,7 @@ export default class SearchMenu extends Component {
 
   @action
   close() {
-    if (this.args?.onClose) {
-      return this.args.onClose();
-    }
+    this.args?.onClose?.();
```

Comment:
This is a change in behaviour. Previously this function would return if `onClose` is passed. Now, it will continue executing the next two lines.

Is that intentional?